recipe/build.sh

```diff
@@ -187,15 +112,19 @@ if [[ ${HOST} =~ .*linux.* ]]; then
                 -nomake tests \
                 -verbose \
                 -skip wayland \
+                -skip qtwebengine \
+                -gstreamer 1.0 \
                 -system-libjpeg \
                 -system-libpng \
                 -system-zlib \
                 -system-sqlite \
                 -plugin-sql-sqlite \
                 -plugin-sql-mysql \
                 -plugin-sql-psql \
+                -xcb \
+                -xcb-xlib \
+                -bundled-xcb-xinput \
```

Comment:
Does this mean we are bundling `xcb-input`?

Reply:
Right now, yes, I'll try reverting it, that flag most probably ended up because of Qt documentation